sim/random-battles.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * A chat plugin to store, calculate, and view winrates in random battle formats.
+ * @author mia-pi-git
+ */
```

Comment:
This looks specific to CGT? Calling it random-battles makes me think it's a randbats thing. Anyway I think @KrisXV wanted to just get rid of CGT since it's not used anymore?